feat: PDF Generation for Chats

monikatrix opened the pull request:
Added JavaScript execution and PDF creation

monikatrix committed:
Add files via upload

Added JavaScript execution and PDF creation

Blank-09 commented:
Sorry the `src` folder is duplicated inside another `src` folder. Please do fix this structure.

monikatrix committed:
feat: PDF Generation for Chats

monikatrix committed:
Revert "Add files via upload"

This reverts commit 709d67f87adb63f35e38195d6bbe843ed6a06132.

Blank-09 committed:
Merge branch 'main' into main

Blank-09 committed:
Import fixed and code formatted

The Files were imported from `com.google.common.io`. Now change to `java.nio.file` package.

Blank-09 reviewed (commented):
All good here and Ready for merge 🎉
